### PR TITLE
[Quick-fix] Deprecate svg, woff, woff2

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: styles/_custom.scss [hunk 1]
```diff
@@ -1380,12 +1380,12 @@ main.main-sidebar {
   font-family: "Material Icons";
   font-style: normal;
   font-weight: 400;
-  src: url("$assets-materialicons-regular-eot"); /* For IE6-8 */
+  // src: url("$assets-materialicons-regular-eot"); /* For IE6-8 */
```
Comment (wengkhink) [inline styles/_custom.scss:1383]: Problem start here

**Comment by ghost** [comment]
I think `svg, woff, woff2` has been causing problem for us. So we are removing it since ttf with eot has wide support for all browser (inclusive of ie). https://medium.com/@aitareydesign/understanding-of-font-formats-ttf-otf-woff-eot-svg-e55e00a1ef2

This is in response of https://jira.configura.com/browse/DOC-375 icon issue (Look comment section)

**Comment by ghost** [comment]
@zarifous @yewch  Please code review !!!

**Comment by zarifous** [comment]
looks good!

**Pull request merged**